<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-6UIHJ6U" ignored="0" name="regression" tests="4" failures="1" timestamp="08 Jan 2020 03:09:40 GMT" time="0.127" errors="0">
  <testcase name="secondMethod" time="0.007" classname="tdd.NewTest">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
at tdd.NewTest.secondMethod(NewTest.java:40)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- secondMethod -->
  <testcase name="m1" time="0.038" classname="tdd.Test1"/>
  <testcase name="firstMethod" time="0.016" classname="tdd.NewTest"/>
  <testcase name="aMethod" time="0.003" classname="tdd.NewTest"/>
</testsuite> <!-- regression -->
